js_machine_front/src/components/app/app.tsx

```diff
@@ -1,13 +1,13 @@
 import React from 'react';
 import './styles/app.css';
-import { NavBar } from 'navBar/navBar';
-import { Partners } from 'partnersLogo/partners';
-import { Events } from 'communityEvents/communityEvents';
-import { News } from 'news/news';
-import { About } from 'about/about';
-import { Main } from 'main/main';
-import { ErrorBoundary } from 'navBar/error-boundary/error-boundary';
-import { Auth } from 'authorization/authorization';
+import { NavBar } from 'components/navBar/index';

```

Comment:
you could import import { NavBar } from 'components/navBar'
not needed define importing from 'index' implicitly.
remove index from all imports